### ISSUE TITLE
Admin Console does not work after upgrading from 18.0.2 to 19.0.1

### ISSUE THREAD

**Issue by westmc** [keycloak/keycloak#14048]
### Describe the bug

We upgraded from 18.0.2 to 19.0.1 and can no longer access the admin console. We believe that KC_HTTP_RELATIVE_PATH seems to no longer be honored.

```
FROM quay.io/keycloak/keycloak:19.0.1 as builder

ENV KC_DB=postgres \
    KC_HTTP_RELATIVE_PATH=/auth \
    KC_HEALTH_ENABLED=true
RUN /opt/keycloak/bin/kc.sh build

FROM quay.io/keycloak/keycloak:19.0.1
COPY --from=builder /opt/keycloak/lib/quarkus/ /opt/keycloak/lib/quarkus/
COPY themes /opt/keycloak/themes
WORKDIR /opt/keycloak
ENTRYPOINT ["/opt/keycloak/bin/kc.sh", "--spi-login-protocol-openid-connect-legacy-logout-redirect-uri=true", "start"]
```

### Version

19.0.1

### Expected behavior

Requests to /auth/admin previously returned the admin console for all versions <=18.0.2

### Actual behavior

We now receive an error page

![image](https://user-images.githubusercontent.com/23019713/186957394-25e0ed09-bcdb-4ca6-adb6-a078e3797ac2.png)

### How to Reproduce?

Change the relative path to /auth and attempt to access /auth/admin

### Anything else?

We have keycloak running as a docker container behind a reverse proxy, but if I use curl to attempt to access the container directly, I get a successful response using /admin which is why I believe the issue is that KC_HTTP_RELATIVE_PATH is not being honored

**Comment by pedroigor** [comment]
@westmc I'm not sure what might be happening. If you can provide a docker-compose reproducing your deployment I can look at it.

The admin console should be accessible from `/auth/admin` when the relative path is set to `/auth`.

**Comment by westmc** [comment]
@pedroigor here is a simplified dockerfile that reproduces the issue. Our config is unchanged from 18

```
FROM quay.io/keycloak/keycloak:19.0.1 as builder

ENV KC_DB=dev-mem \
    KC_HTTP_RELATIVE_PATH=/auth \
    KC_HEALTH_ENABLED=true
RUN /opt/keycloak/bin/kc.sh build

FROM quay.io/keycloak/keycloak:19.0.1
COPY --from=builder /opt/keycloak/lib/quarkus/ /opt/keycloak/lib/quarkus/
WORKDIR /opt/keycloak
ENTRYPOINT ["/opt/keycloak/bin/kc.sh", "--spi-login-protocol-openid-connect-legacy-logout-redirect-uri=true", "start"]
```

Here are the runtime env variables...

```
 -e KEYCLOAK_ADMIN=foo \
 -e KEYCLOAK_ADMIN_PASSWORD=bar \
 -e KC_HOSTNAME_STRICT=false \
 -e KC_HOSTNAME_STRICT_HTTPS=false \
 -e KC_HTTP_ENABLED=true \
 -e KC_PROXY=edge \
```

Yes, you are right about the 404 page. Sorry, I thought that was coming from keycloak, but it was the proxy. This is what I get from keycloak

```
curl -v http://keycloak-test:8080/auth/admin
*   Trying 172.100.0.26...
* TCP_NODELAY set
* Connected to keycloak-test (172.100.0.26) port 8080 (#0)
> GET /auth/admin HTTP/1.1
> Host: keycloak-test:8080
> User-Agent: curl/7.61.1
> Accept: */*
>
< HTTP/1.1 404 Not Found
< Referrer-Policy: no-referrer
< X-Frame-Options: SAMEORIGIN
< Strict-Transport-Security: max-age=31536000; includeSubDomains
< X-Content-Type-Options: nosniff
< X-XSS-Protection: 1; mode=block
< Content-Type: application/json
< content-length: 103
<
* Connection #0 to host keycloak-test left intact
{"error":"RESTEASY003210: Could not find resource for full path: http://keycloak-test:8080/auth/admin"}
```

**Comment by westmc** [comment]
@lexcao I tried your suggestions and it started working. Based on some additional testing, it looks like KC_HTTP_RELATIVE_PATH changed from a build option to a runtime option in 19. The [config documentation](https://www.keycloak.org/server/all-config) should probably be updated. I was able to get it to work by removing the flag from the dockerfile entirely and just setting the flag as part of the run command.

The change in the dockerfile to copy the full folder should probably be added to the release notes for 19. I got the original dockerfile from [here](https://www.keycloak.org/server/containers) and I noticed that it has been updated to what you suggested.

FYI @pedroigor
